src/Header/Header.css

```diff
@@ -18,5 +18,13 @@
   right: 10px;
   top: 30px;
   width: 4rem;
+  background-color: antiquewhite;
```

Comment:
Since the login button is now a anchor tag I began styling the link to look like a button